roles/weave/tasks/main.yml

```diff
@@ -27,14 +30,18 @@
     - weave
 
 - name: bring up weave bridge
-  shell: ifup weave
+  command: ifup weave
   sudo: yes
   tags:
     - weave
 
 - name: configure weave bridge for docker
   sudo: yes
-  lineinfile: "dest=/etc/default/docker state=present regexp=^DOCKER_OPTS= line='DOCKER_OPTS=\"{{ weave_docker_opts }}\"'"
+  lineinfile:
+   dest: /etc/default/docker
+   state: present
+   regexp: ^DOCKER_OPTS=
+   line: DOCKER_OPTS=\"{{ weave_docker_opts }}\"
```

Comment:
Will this still work if we add docker options in other areas e.g. private dockerhub, storage options etc.